libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/resources/testProject/new-mpp-android-source-sets/lib/build.gradle.kts

```
@@ -0,0 +1,38 @@
+plugins {
+    id("com.android.library")
+    kotlin("multiplatform")
+}
+
+android {
+    compileSdkVersion(28)
+    defaultConfig {
+        minSdkVersion(21)
+        targetSdkVersion(28)
+        testInstrumentationRunner = "android.support.test.runner.AndroidJUnitRunner"
+    }
+}
+
+kotlin {
+    android()
+    macosX64("macos")
+
+    sourceSets {
+        getByName("commonMain").dependencies {
+            implementation(kotlin("stdlib-common"))
+        }
+
+        getByName("commonMain").dependencies {
```

Looks like there's a mistake there.
```suggestion
        getByName("commonTest").dependencies {
```

---

Thanks a lot! Was indeed a mistake 🤷  ☺️